src/bioregistry/data/bioregistry.json

```diff
@@ -5259,6 +5259,76 @@
       "orcid": "0000-0003-4423-4370"
     }
   },
+  "asteraceaegd.genome": {
+    "contact": {
+      "email": "greatchen@cdutcm.edu.cn",
+      "name": "Wei Chen",
+      "orcid": "0000-0003-0214-3032"
```

Comment:
I'm actually not sure about the ORCID here. The affiliations don't exactly match. @nagutm could you take another look if this is the same person as the author of this paper?

Reply:
Actually, I'm pretty sure this is the right ORCID: https://orcid.org/0000-0002-6857-7696

Reply:
I mixed up the Chengdu University of Technology and Chengdu University of Traditional Chinese Medicine. Thanks for catching that, I will update the record